Nodejs/Product/Nodejs/NpmUI/NpmInstallWindowResources.Designer.cs

```diff
@@ -250,7 +259,7 @@ public class NpmInstallWindowResources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Package list.
+        ///   Looks up a localized string similar to Packages.
```

Comment:
"Search Results" could also work

Reply:
I don't really have preference. Unless you feel really strong about it, I can submit another PR for that.